src/core/src/op/util/avg_pool_base.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/util/avg_pool_base.hpp"
+
+#include "itt.hpp"
+
+ov::op::util::AvgPoolBase::AvgPoolBase(const Output<Node>& arg,
```

Comment:
This is current approach in `src/core`. There are still files with "older" nesting style, what might be confusing.
Apply it respectively to other definitions below.
```suggestion
namespace ov {
namespace op {
namespace util {
AvgPoolBase::AvgPoolBase(const Output<Node>& arg,
```

Reply:
Applied in the https://github.com/openvinotoolkit/openvino/pull/23483/commits/0fa32444523ffe467e6cdc7d1321ad0e35c32d08
Thanks